tests/tests.rs

```diff
@@ -1059,6 +1067,16 @@ sherlock!(feature_89_files_with_matches, "Sherlock", ".",
 });
 
 // See: https://github.com/BurntSushi/ripgrep/issues/89
+sherlock!(feature_89_files_without_matches, "Sherlock", ".",
```

Comment:
I actually meant issue #89, since the test is about the interaction between `--null` and this feature.